cordova-fetch/spec/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Hmm. Maybe just name it differently. My concern is that presence of another `package.json` might be a bit confusing + break some tools in some scenarios